test/unit/server-acl-init-job.bats

```diff
@@ -185,20 +179,21 @@ load _helpers
 #--------------------------------------------------------------------
 # aclBindingRuleSelector/global.acls.manageSystemACLs
 
-@test "serverACLInit/Job: no acl-binding-rule-selector flag by default" {
+@test "serverACLInit/Job: acl-binding-rule-selector flag set by default" {
   cd `chart_dir`
   local actual=$(helm template \
-      -x templates/server-acl-init-job.yaml \
+      -s templates/server-acl-init-job.yaml \
+      --set 'global.acls.manageSystemACLs=true' \
       --set 'connectInject.aclBindingRuleSlector=foo' \
```

Comment:
Hmm, this test implies that we shouldn't be setting aclBindingRuleSelector. It's strange that it's passing 🤔

Reply:
Oh there's a typo in `aclBindingRuleSlector`, that's why it works!

Reply:
nice catch, I deleted the `--set 'connectInject.aclBindingRuleSlector=foo' \`